<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReferencesResultDialog</class>
 <widget class="QWidget" name="FindReferencesResultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find References</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="listLabel">
     <property name="text">
      <string>References:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="referenceList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelBn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="jumpToBn">
       <property name="text">
        <string>Jump To</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBn</sender>
   <signal>clicked()</signal>
   <receiver>FindReferencesResultDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
